### PR TITLE
Remove Click and use jQuery handler.

### PR DESCRIPTION
**Pull request by RCAVelez**
Did this to avoid having to see which html tags call the message function,
Create separation of concerns within template,
And Easier testing.

### CODE REVIEW

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 1]
```diff
@@ -99,10 +99,9 @@
       <input type="text" name="" placeholder="First Name">
       <input type="text" name="" placeholder="Last Name">
       <input type="text" name="" placeholder="Email">
-      <button type="button" name="button" onclick="message()">Send</button>
+      <button class="form-button" type="button" name="button">Send</button>
```
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:98]: Please do the same for the other 6 `onclick="animateScrollTop..."`

**Review by genedwards — CHANGES_REQUESTED** [review]

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html (hunk 1 above)
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:98]: Done, did for every a link

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js [hunk 2]
```diff
@@ -19,7 +19,15 @@ const animateScrollTop = (targetClass) => {
...
-  $(".form-message").css({'visibility':'visible'})
-}
 
+$(document).ready(function() {
+  $(".form-button").click(function() {
+    $(".form-message").css("visibility", "visible")
+  });
+
+  $('a').click(function() {
+    console.log($(this).attr('href'))
```
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js:29]: are you keeping this console.log?

**Review by genedwards — APPROVED** [review]

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js (hunk 2 above)
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js:29]: oh whoops, I removed it

**Pull request merged**